docs/integration/mongodb.md

```diff
@@ -116,3 +64,162 @@ update | mongo.ops.update | Sum | Long | Update operations
 delete | mongo.ops.delete | Sum | Long | Delete operations
 getmore | mongo.ops.getmore | Sum | Long | Getmore operations
 getmore | mongo.replica.ops.getmore | Sum | Long | Getmore operations
+mongo.active_reads | mongo.active_reads | Avg | long |
```

Comment:
No descriptions starting from this row?

Reply:
Difficult to find, painful to add, almost no value...